public/app/features/teams/UserTeamsList.tsx

```diff
@@ -0,0 +1,88 @@
+import { css } from '@emotion/css';
+import React, { useEffect, useMemo } from 'react';
+import { connect, ConnectedProps } from 'react-redux';
+
+import { GrafanaTheme2 } from '@grafana/data';
+import { CellProps, InteractiveTable, Icon, Column, Avatar, useStyles2 } from '@grafana/ui';
+import { Stack } from '@grafana/ui/src/unstable';
+import { StoreState, Team } from 'app/types';
+
+import { Page } from '../../core/components/Page/Page';
+import { TableWrapper } from '../admin/Users/TableWrapper';
+
+import { loadUsersTeams } from './state/actions';
+
+type Cell<T extends keyof Team = keyof Team> = CellProps<Team, Team[T]>;
```

Comment:
I'd just export this type from the `TeamList` component or move it to a separate `types.ts` file inside the Team folder to not duplicate it.